### ISSUE TITLE
Alt keys and compatibility with vim

### ISSUE THREAD

**Issue by elebeaup** [onivim/oni2#2963]
According the vim documentation, the notation <A-...> and <M-...> must have the same behavior :
```
<M-…>		alt-key or meta-key		META ALT <M-
<A-…>		same as <M-…>	        <A-
```

But the following mapping `nnoremap <A-j> :m .+1<CR>==` works but not the mapping `nnoremap <M-j> :m .+1<CR>==`.
I have the following error :
```
[INFO]  [ 8.073s] Oni2.Store.dispatch : dispatch: (Notification
   (Created
      { id = 1; kind = Error;
        message =
        "Error mapping <M-j> to :m .+1<CR>==: Unrecognized modifier:m- in: <M-j>";
        source = None; yOffsetAnimation = <opaque> }))
[DEBUG] [ 8.075s] Oni2.Store.dispatch : After: (Notification
   (Created
      { id = 1; kind = Error;
        message =
        "Error mapping <M-j> to :m .+1<CR>==: Unrecognized modifier:m- in: <M-j>";
        source = None; yOffsetAnimation = <opaque> }))
```

So, is it an issue or the normal behavior ?

**Comment by bryphe** [comment]
Thanks for logging the issue @elebeaup ! It is indeed a bug - we should handle `M-` the same was as Vim.

**Comment by bryphe** [comment]
This should be fixed now in #2978 - we now handle `m-` the same as `a-`.
